spec/support/features/session_helper.rb

```diff
@@ -266,6 +266,14 @@ def click_continue
       click_button t('forms.buttons.continue') if page.has_button?(t('forms.buttons.continue'))
     end
 
+    def click_agree_and_continue(allow_missing: false)
+      if allow_missing
```

Comment:
The `click_continue` above only sometimes clicks continue...which makes refactoring a challenge

I decided I wanted to mimimize the "click if it's not there behavior", so I parameterized it, and there's only one spec (signing in a second time) that used this. It seemed not quite worth the trouble to push the behavior into the caller for this case.... but open to feedback here

Reply:
I like

Reply:
Actually I just realized, I can make a second method, instead of this complex parameter....going to do that instead